pkg/network/base_connection.go

```diff
@@ -118,7 +129,7 @@ func (t *baseTcpConnection) consumeConnection(conn net.Conn) {
 				}
 				err := t.write(conn, msg)
 				if err != nil {
-					t.outboundMessageChannel <- msg
+					t.WriteMessage(t.ctx, msg) // TODO
```

Comment:
TODO?

Reply:
yeah, this returns an error, but it's not really clear whether this error should be used or not. I plan to solve this issue properly with #29, that will just add a writing queue, replacing that channel

Reply:
OK, thanks